system/keyboardlayouts/german.xml

```diff
@@ -6,19 +6,19 @@ Default font lacks support for all characters
 <keyboardlayouts>
   <layout language="German" layout="QWERTZ">
     <keyboard>
-      <row>1234567890</row>
+      <row>1234567890ß</row>
       <row>qwertzuiopü</row>
       <row>asdfghjklöä</row>
-      <row>yxcvbnm</row>
+      <row>yxcvbnmμ</row>
```

Comment:
"μ" should not be there IMO, rather move it to the symbols (I didn't even know it's on my keyboard ;) )